samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -2608,11 +2608,11 @@ mfxStatus CmdProcessor::VerifyAndCorrectInputParams(TranscodingSample::sInputPar
         return MFX_ERR_UNSUPPORTED;
     }
 
-    if (InputParams.nLADepth && (InputParams.nLADepth < 10))
+    if (InputParams.nLADepth && (InputParams.nLADepth > 100))
```

Comment:
Did you intend to write (mind `!` and `||`):
`if (!InputParams.nLADepth || (InputParams.nLADepth > 100))`

Mind that after your change the condition @ line 2613 stop to make sense. I mean this one:
`if ((InputParams.nLADepth != 1) || (!InputParams.nMaxSliceSize))`

Please, check its correctness and update/remove as needed. Question: does `-mss` require `nLADepth == 1` or it can work with other depths as well?